doc/fping.pod

```diff
@@ -155,6 +155,12 @@ to any target (default is 10, minimum is 1).
 
 Set the interface (requires SO_BINDTODEVICE support).
 
+=item B<--icmp-timestamp>
+
+Send ICMP timestamp requests (ICMP type 13).
+If B<fping> not recive ICMP timestamp reply, "(Timestamp unknown)" is returned.
+IPv4 only, requires root privileges.
```

Comment:
Perhaps insert something like the following?

`Cannot be used together with B<-b> because ICMP timestamp messages have a fixed size.`